Sources/AppearanceManager.swift

```diff
@@ -50,3 +55,10 @@ extension UINavigationController {
         return self.topViewController
     }
 }
+
+extension UISwitch {
+    @objc dynamic var cornerRadius: CGFloat {
+        get { return layer.cornerRadius }
+        set { layer.cornerRadius = newValue }
+    }
+}
```

Comment:
I personally find introducing this more confusing than helpful

Reply:
🤔 would it be better to just set the cornerRadius locally wherever a switch is used?

Reply:
Something that you could do, which I would understand is this: 
``class VLCSwitch: UISwitch {
  var OffColor: UIColor? {
        didSet {
            self.layer.cornerRadius = 16 
            self.backgroundColor = OffColor
        }
    }
}

Reply:
I prefer the above approach as well but InAppSettingsKit is not letting me do this :(
They force cast `UISwitch` to their `IASKSwitch` in their library.